packages/palette/src/elements/Image/Image.tsx

```diff
@@ -7,12 +7,28 @@ import {
 } from "./Image.shared"
 import { LazyImage } from "./LazyImage"
 
-/** Image */
-export const Image = ({ lazyLoad = false, ...props }: ImageProps) => (
-  <LazyImage preload={!lazyLoad} imageComponent={BaseImage} {...props} />
-)
+/** Props for a web-only Image component. */
+interface WebImageProps extends ImageProps {
+  preventRightClick?: boolean
+}
+
+/** A web-only Image component. */
```

Comment:
It might just be a good idea to put a comment banner at the top of this file. Everything in the file is web only.

Reply:
Leaving this comment is fine though!

Reply:
Also that can happen in a follow up.